### PR TITLE
[new] add bootstrap-fileinput@4.2.6 with git auto-update, close #5182

### LINKED ISSUE #5182: [Request] Add bootstrap-fileinput
**Library name:** bootstrap-fileinput
**Git repository url:** https://github.com/kartik-v/bootstrap-fileinput
**npm package url(optional):** 
**Official homepage:** https://github.com/kartik-v/bootstrap-fileinput
**Wanna say something? Leave message here:**

### PR DESCRIPTION
**Pull request by Amomo**
Hi @LeaYeh , I add a new lib with git auto-update from the request #5182
repo: https://github.com/kartik-v/bootstrap-fileinput
Additionally, I minify the `fileinput_locale_*.js` files, and record them in "requiredFile" field.
Would you mind checking this PR for me? Thank you~ :-)

### CODE REVIEW

**Comment by LeaYeh** [comment]
do we need all files in `/js` ?

**Comment by Amomo** [comment]
@LeaYeh because the lib provide a UI, it needs these `locale_*` files to change UI's language for users from all over the world. I think we should collect them.

**Comment by LeaYeh** [comment]
ok got it. @PeterDaveHello

File: ajax/libs/bootstrap-fileinput/package.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    "source": "git",
+    "target": "git://github.com/kartik-v/bootstrap-fileinput.git",
+    "basePath": "",
+    "files": [
+      "js/fileinput*.js",
+      "css/fileinput*.css"
+    ]
+  },
+  "requiredFile": [
+    "js/fileinput_locale_*.min.js"
```
Comment (PeterDaveHello) [inline ajax/libs/bootstrap-fileinput/package.json]: Should not use wildcard here.

**Comment by Amomo** [comment]
@LeaYeh I have done the correction.
Could you come to check this PR? Thank you~ :-D

**Comment by LeaYeh** [comment]
@Amomo  could you tell me what Peter means?

**Comment by Amomo** [comment]
@LeaYeh 
He means that I should not use `*` in "requiredFile".
I need to list every required files clearly.

**Comment by Amomo** [comment]
Hi @Piicksarn would you mind checking this PR for me?
Thank you~ :-D

**Comment by Amomo** [comment]
@LeaYeh , @Piicksarn I have rebase this commit to match latest master branch.
Would you mind checking this PR for me? Thank you~ :-D

**Comment by Amomo** [comment]
@LeaYeh , @Piicksarn I have rebase this commit to match latest master branch.
Would you mind checking this PR for me? Thank you~ :-D

**Comment by Amomo** [comment]
@LeaYeh , @Piicksarn Would you mind checking this PR for me? Thank you~ :-D

**Comment by Amomo** [comment]
@Piicksarn Thanks for your reminding~
Would you mind checking this PR for me? Thank you!

**Comment by Piicksarn** [comment]
@PeterDaveHello, would you please take a look at the pr? Thank you!

**Comment by PeterDaveHello** [comment]
Let's use `repository` instead of `repositories`.

**Comment by Amomo** [comment]
@Piicksarn I have modified the repository field, could you check it again?
Thank you! >_<

**Comment by Piicksarn** [comment]
@Amomo, sorry for checking late...
The pr need rebase to master~ ~

**Comment by Amomo** [comment]
@Piicksarn I have rebased it, could you check it again?
Thank you very much~~ :-D

**Comment by Piicksarn** [comment]
@PeterDaveHello, would you mind to have a look at the pr?
thank you!

**Comment by PeterDaveHello** [comment]
missing img ...

**Comment by Amomo** [comment]
Thanks for your reminding, Peter 😭
@Piicksarn bootstrap-fileinput has a new version `v4.2.6` now, could you please check this PR again?
Thank you!

**Comment by Piicksarn** [comment]
@PeterDaveHello, would you please take a look at the commit ?
thank you!

**Pull request merged**